HttpMockPlayer/Player.cs

```diff
@@ -272,6 +279,20 @@ internal IEnumerable<(string Property, object ThisValue, object OtherValue)> Get
                 }
             }
 
+            private static string NormalizeXml(string content)
+            {
+                var xml = XElement.Parse(content);
+
+                // CG: Namespace declarations may be sorted differently, ignore them
+                // https://stackoverflow.com/questions/20701750/c-sharp-xml-serialization-order-of-namespace-declarations
+                foreach (var attr in xml.Attributes().Where(a => a.IsNamespaceDeclaration).ToArray())
+                {
+                    attr.Remove();
```

Comment:
I think it's possible to compare XML as XML objects, not as strings, and they should be equal disregard of attributes order. Otherwise there might be other attributes that appear in random order..

Reply:
I tried comparing the XML but it failed, possibly because it actually compares the order of attributes. This was the easiest solution I could find, but it might be possible to do some more advanced comparison.